tasks/queue-granules/index.js

```diff
@@ -81,7 +81,22 @@ async function queueGranules(event) {
         granuleBatch[0].dataType,
         granuleBatch[0].version
       );
-      const executionArn = enqueueGranuleIngestMessage({
+      await pMap(
+        granuleBatch,
+        (queuedGranule) => granulesApi.updateGranule({
+          prefix: event.config.stackName,
+          body: {
+            collectionId: constructCollectionId(
+              queuedGranule.dataType,
+              queuedGranule.version
+            ),
+            granuleId: queuedGranule.granuleId,
+            status: 'queued',
+          },
```

Comment:
Since this body does not include execution information, do we actually have any situation where we send a `queued` update with execution info?